src/toucan/db.clj

```diff
@@ -550,8 +599,8 @@
   {:style/indent 1}
   [model & options]
   (simple-select model (where+ {:select (or (model->fields model)
-                                             [:*])}
-                                options)))
+                                            [:*])}
+                               options)))
```

Comment:
[parinfer](https://shaunlebron.github.io/parinfer/) really doesn't like those spaces

Reply:
yeah not sure how they got indented that way but thanks for fixing it